_security-analytics/threat-intelligence/api/source.md

```diff
@@ -0,0 +1,482 @@
+| `description`  | String     | A description of the IOC.    |
+| `labels`   | Array of Strings | Any labels or tags associated with the IOC.  |
+| `feed_id`   | String           | A unique identifier for the feed to which the IOC belongs.    |
+| `spec_version` | String           | The specification version used for the IOC.    |
+| `version`      | Integer    | A version number for the IOC.    |
+
+### Response fields
+
+| Field     | Data type   | Description   |
+| :---- | :--- |:----- |
+| `_id`     | String    | The unique identifier for the threat intelligence source.     |
+| `_version`  | Integer           | The version number of the threat intelligence source.   |
+| `source_config`    | Object   | The configuration details of the threat intelligence source.    |
+| `source_config.name`    | String    | The name of the threat intelligence source.   |
+| `source_config.format`   | String     | The format of the threat intelligence data.    |
+| `source_config.type`   | String   | The type of the threat intelligence source.   |
+| `source_config.ioc_types`  | Array of Strings  | The types of IoCs supported by the source.   |
+| `source_config.description`   | String  | The description of the threat intelligence source.  |
+| `source_config.created_by_user`  | String or Null    | The user who created the threat intelligence source.    |
+| `source_config.created_at`    | String (DateTime) | The date and time when the threat intelligence source was created.      |
+| `source_config.source`  | Object   | Contains details about the source of the threat intelligence data.   |
+| `source_config.source.ioc_upload`    | Object    | The details about the IoC upload.   |
+| `source_config.source.ioc_upload.file_name` | String   | The name of the file uploaded. Example: `test`. |
+| `source_config.source.ioc_upload.iocs`      | Array of Objects  | Any additional information about the IoC upload. When the IoC is stored successfully, this appears as an empty array.   |
+| `source_config.enabled`   | Boolean    | Indicates whether the threat intelligence source is enabled.  |
+| `source_config.enabled_time`    | String or Null    | The date and time when the source was enabled.   |
+| `source_config.last_update_time`  | String (DateTime) | The date and time when the threat intelligence source was last updated.  |
+| `source_config.schedule`  | String or Null    | The schedule for the threat intelligence source.  |
+| `source_config.state`    | String    | The current state of the threat intelligence source.  |
+| `source_config.refresh_type`    | String   | The type of refresh applied to the source.  |
+| `source_config.last_refreshed_user`   | String or Null    | The user who last refreshed the source. |
+| `source_config.last_refreshed_time`         | String (DateTime) | The date and time when the source was last refreshed. |
+
+### Example requests 
+
+
```

Comment:
```suggestion
The following examples show how to use the Source API.
```

Reply:
@Naarcha-AWS "example requests show you how"